src/vconsole/vconsole-setup.c

```diff
@@ -295,6 +301,7 @@ int main(int argc, char **argv) {
         /* Let the kernel command line override /etc/vconsole.conf */
         if (detect_container() <= 0) {
                 r = parse_env_file("/proc/cmdline", WHITESPACE,
+                                   "vconsole.vc.max", &vc_cnt_s,
```

Comment:
the name of the kernel cmdline option should really be in sync with the field name in vconsole.conf